05-onchain.md

```diff
@@ -588,25 +590,25 @@ Note: even if the `to_remote` output is not swept, the resulting
 
 # Generation of HTLC Transactions
 
-If `option_anchors` does not apply to the commitment transaction, then
-HTLC-timeout and HTLC-success transactions are complete transactions with
-(hopefully!) reasonable fees and must be used directly.
+If `SIGHASH_ALL` is used for HTLC transactions, then HTLC-timeout and
+HTLC-success transactions are complete transactions with (hopefully!)
+reasonable fees and must be used directly.
```

Comment:
This section describes *when* to use `SIGHASH_ALL` vs `SIGHASH_SINGLE|SIGHASH_ANYONECANPAY`.  So we shouldn't assume the reader already knows which sighashes to use for each channel type.

```suggestion
If `option_anchors` and `zero_fee_commitments` do not apply, then HTLC-timeout and
HTLC-success transactions are signed with SIGHASH_ALL and are complete transactions with (hopefully!)
reasonable fees and must be used directly.
```